### PR TITLE
Ethan

### PR DESCRIPTION
**Pull request by VanguardAWS**

### CODE REVIEW

**Review by brucestull — CHANGES_REQUESTED** [review]
[Lab 9: ATM](https://github.com/PdxCodeGuild/class_swordfish/blob/main/1%20Python/labs/09%20ATM.md)
**Needs Work**
* Comments:
    * Interest is not calculating properly:
        * Interest on `100`:
            * Expected: `.1`
            * But result is `1`

**Review by brucestull — CHANGES_REQUESTED** [review]
https://github.com/PdxCodeGuild/class_swordfish/pull/113#pullrequestreview-1120895019

File: code/ethan/python/lab09.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
+#  ATM
+
+import math
+
+class ATM():
+    def __init__(self):
+        self.balance = 0    
+        self.interest = .01
```
[the remark this answers is not in the document]
Comment (VanguardAWS) [reply code/ethan/python/lab09.py]: lab09 fixed

**Review by brucestull — APPROVED** [review]
[Lab 9: ATM](https://github.com/PdxCodeGuild/class_swordfish/blob/main/1%20Python/labs/09%20ATM.md)
**Completed**

**Pull request merged**